config.template.yml

```
@@ -93,6 +93,13 @@ authentication_backend:
   # than one instance and therefore is recommended for
   # production.
   ldap:
+    # The LDAP implementation, this affects elements like the attribute utilised for resetting a password.
+    # Acceptable options are as follows:
+    # - 'activedirectory' - For Microsoft Active Directory.
+    # - 'custom' - For custom specifications of attributes and filters.
+    # This currently defaults to 'custom' to maintain existing behaviour.
+    # implementation: custom
```

```suggestion
    implementation: custom
```

@james-d-elliott, is it worth with the defaults now we uncomment this and in turn comment the attributes/filters which have a default specified for the custom implementation and only leave the `n/a` ones uncommented.
Perhaps also with an updated description in the config templates which point back to the docs re: defaults and the ability to override?